ee/modules/650-runtime-audit-engine/images/rules-loader/Dockerfile

```
@@ -0,0 +1,7 @@
+ARG BASE_SHELL_OPERATOR
+FROM $BASE_SHELL_OPERATOR
+COPY hooks/ /hooks
+COPY requirements.txt requirements.txt
+
+RUN apk add --no-cache curl python3 py3-pip && \
+    pip3 install -r requirements.txt
```

```suggestion
COPY requirements.txt requirements.txt

RUN apk add --no-cache curl python3 py3-pip && \
    pip3 install -r requirements.txt

COPY hooks/ /hooks
```

Not a big deal, just a small speed up for rebuilds.